store-implementation/federated-store/src/main/java/uk/gov/gchq/gaffer/federatedstore/operation/IFederationOperation.java

```diff
@@ -32,8 +32,8 @@ public interface IFederationOperation extends Operation {
     boolean isUserRequestingAdminUsage();
 
     @JsonGetter("userRequestingAdminUsage")
```

Comment:
I think this was being done to ensure that FederationOperations did not show the `userRequestingAdminUsage` field in json when it was false, probably as this is default. I think a test should be added to assert this behaviour and another method should be used to ensure this passes. I can look at this next week

Reply:
Changing to use `@JsonInclude(value = JsonInclude.Include.NON_DEFAULT)` removing the need for the `null` return and has the same effect. This is because globally `null` is not included already. This also allows the extra method to be removed entirely.

There is a test (in `FederatedOperationTest`) which unintentionally checks that this field (and also `skipFailedFederatedExecution`) is not present.